framework/include/tinyalsa/tinyalsa.h

```diff
@@ -257,19 +257,39 @@ struct pcm;
  * @param[in] flags  Specify characteristics and functionality about the pcm.
  * @param[in] config The hardware and software parameters to open the PCM with.
  * @returns On success, A PCM structure returned. On failure, invalid PCM structure returned.
- * @since Tizen RT v1.1
+ * @since Tizen RT v1.x
  */
 struct pcm *pcm_open(unsigned int card, unsigned int device, unsigned int flags, const struct pcm_config *config);
 
 /**
+* @brief Stops a PCM. Any data present in the buffers will be dropped.
+*
+* @details @b #include <tinyalsa/tinyalsa.h>
+* @param[in] A PCM handle.
+* @returns On success, zero; on failure, a negative number.
+* @since Tizen RT v1.x
+*/
+int pcm_drop(struct pcm *pcm);
+
+/**
+* @brief Stop a PCM preserving pending frames.
+*
+* @details @b #include <tinyalsa/tinyalsa.h>
+* @param[in] pcm A PCM handle.
+* @returns On success, zero; on failure, a negative number.
+* @since Tizen RT v1.x
+*/
+int pcm_drain(struct pcm *pcm);
+
+/**
  * @brief Opens a PCM by it's name.
  *
  * @details @b #include <tinyalsa/tinyalsa.h>
  * @param[in] name   The name of the PCM.
  * @param[in] flags  Specify characteristics and functionality about the pcm.
  * @param[in] config The hardware and software parameters to open the PCM with.
  * @returns On success, A PCM structure returned. On failure, NULL or invalid PCM structure returned.
- * @since Tizen RT v1.1
+ * @since Tizen RT v1.x
```

Comment:
please roll back 1.1 for current api, apply v1.x only for new apis.

Reply:
Done